stable/aws-for-fluent-bit/values.yaml

```diff
@@ -116,10 +116,10 @@ serviceAccount:
 
 resources:
   limits:
-    memory: 500Mi
+    memory: 150Mi
```

Comment:
I strongly agree with lowering the request. Lowering the limit is more dangerous. Putting it at 250 would be safer for more folks. If you have a lot of logs it can and will use that much memory.

The problem this PR wanted to address was the inability to run this on a small instance- and changing the request should fix that right? but the limit could be kept higher? Or am I talking nonsense.

Reply:
Hey Wesley!

Reply:
Cool! In retrospect I wish I could amend my answer to that question a little bit. It's not abnormal to see memory usage up to a few hundred MB if you are sending a lot of logs or doing a lot of processing. And I think my estimate for the go plugins was a bit off, you can easily see pretty very high memory usage with those. The answer I gave is accurate for the most common cases with the core plugins.

Reply:
Thanks! I've amended the `limits` to now be 250Mi 👍